pkg/common/action.go

```diff
@@ -43,3 +43,33 @@ func (sa *SuggestedActions) DescribeActions() string {
 	}
 	return strings.Join(acts, ", ")
 }
+
+// Simplify simplifies the suggested actions by removing duplicate actions and
+// keeping only the most severe actions.
+// And it also removes the descriptions and references.
+// HW inspection takes priority over reboot.
+func (sa *SuggestedActions) Simplify() *SuggestedActions {
+	simplified := &SuggestedActions{
+		References:   nil,
+		Descriptions: nil,
+	}
+
+	actionsToKeep := make(map[RepairActionType]bool)
+	for _, act := range sa.RepairActions {
+		actionsToKeep[act] = true
+	}
+	if _, ok := actionsToKeep[RepairActionTypeHardwareInspection]; ok {
+		delete(actionsToKeep, RepairActionTypeRebootSystem)
+	}
```

Comment:
We cannot do this since such as xid 79 defined two actions: reboot and hardware-inspection.. XID component sets the action by the history events.. for example, xid component suggests rebooting system for the first and second 79 error, suggests hardware-inspection to the third times...

Reply:
right, let me create a new PR